macaroons/README.md

```diff
@@ -114,3 +114,29 @@ removing all three default macaroons (`admin.macaroon`, `invoice.macaroon` and
 `readonly.macaroon`, **NOT** the `macaroons.db`!) from their
 `data/chain/<chain>/<network>/` directory inside the lnd data directory and
 restarting lnd.
+
+
+## Manage macaroons
+
+As of lnd `(TODO: add version)` there are `listmacaroonids` and
+`deletemacaroonid` available through gPRC and command line.
+Users can view a list of all macaroon root key IDs that are in use using:
+
+`lncli listmacaroonids`
+
+And remove a specific macaroon root key ID using command:
+
+`lncli deletemacaroonid root_key_id`
+
+### Upgrading from `(TODO: add version)` or earlier
```

Comment:
Nothing really changed with the macaroon permissions with this PR. I think it's enough to just have the above `Upgrading from v0.8.0-beta...`.

Reply:
Removed.
Between the version `v0.9.0-beta` and this one, users need to delete the old macaroons  though. Otherwise the old `admin_macaroon` won't have the newly added `macaroon/read` and `macaroon/write` permissions.